<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>V4l2sinkProperties</class>
 <widget class="QDialog" name="V4l2sinkProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>V4l2sinkProperties</string>
  </property>
  <widget class="QCheckBox" name="checkBox_auto">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AutoStart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>411</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
    <property name="spacing">
     <number>25</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="label_device">
      <property name="text">
       <string>Device           </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_dev">
      <property name="placeholderText">
       <string>/dev/video1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
